package.json

```diff
@@ -82,7 +84,7 @@
     "sass-loader": "^7.0.1",
     "storybook-readme": "^5.0.8",
     "style-loader": "^0.20.3",
-    "ts-loader": "^3.5.0",
+    "ts-loader": "^4.0.0",
```

Comment:
I ran into a weird error with Cypress without this change. @lychyi @anicholls Should we upgrade to the latest (6.0.4)?